packages/cli/src/helpers.mts

```diff
@@ -10,6 +10,14 @@ import { has } from 'lodash-es';
 
 const DIR_NAME = getDirNameForCurrentFile(import.meta);
 
+// helpers when when working with url to generate service
+const DEFAULT_BRANCH = 'develop';
+const URL_PREFIX = 'https://api.github.com/repos/harness/harness-core/contents/';
```

Comment:
OATS cli is not specific to harness. `harness/harness-core` can come from env variable, or yaml path.